api/test/server.js

```diff
@@ -0,0 +1,75 @@
+
+var chai = require('chai');
+var assertArrays = require('chai-arrays');
+var expect = chai.expect;
+chai.use(assertArrays);
+var request = require("request");
+
+
+describe("API Server", function(){
+  describe("Story Routes", function() {
+    describe("/story/getStoryById/:id", function() {
```

Comment:
In case you haven't come across it before: javascript [arrow syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) is pretty convenient.

Lets us declutter a bit:
```suggestion
describe("API Server", () => {
  describe("Story Routes", () => {
    describe("/story/getStoryById/:id", () => {
```